### ISSUE TITLE
adding neotest

### ISSUE THREAD

**Issue by luxus** [AstroNvim/astrocommunity#32]
```lua
   {
    "nvim-neotest/neotest",
    config = function()
      -- get neotest namespace (api call creates or returns namespace)
      local neotest_ns = vim.api.nvim_create_namespace "neotest"
      vim.diagnostic.config({
        virtual_text = {
          format = function(diagnostic)
            local message = diagnostic.message:gsub("\n", " "):gsub("\t", " "):gsub("%s+", " "):gsub("^%s+", "")
            return message
          end,
        },
      }, neotest_ns)
      require("neotest").setup {
        -- your neotest config here
        adapters = {
          require "neotest-go",
          require "neotest-rust",
          require "neotest-python",
        },
      }
    end,
    ft = { "go", "rust", "python" },
    dependencies = {
      "nvim-neotest/neotest-go",
      "nvim-neotest/neotest-python",
      "rouge8/neotest-rust",
    },
  },
```
this code should be in the packs i guess.. the additional config = stuff is from the neotest-go site

**Comment by mehalter** [comment]
`FixCursorHold.nvim` can be removed as their PR has been merged into Neovim already and is redundant.

**Comment by luxus** [comment]
fixed by #45
